### PR TITLE
Add names to Qiskit bot

### PR DESCRIPTION
**Pull request by frankharkins**
Adds usernames for each file, escaping based on their notification preferences.

### CODE REVIEW

File: qiskit_bot.yaml [hunk 1]
```diff
@@ -1,30 +1,240 @@
...
-# We use quotes around users who don`t want a GitHub notification,
-# but whom we still want their name in the Qiskit Bot message so
-# people know they are relevant.
+always_notify: true
+notification_prelude: |
+  Thanks for contributing to Qiskit documentation!
+
+  Before your PR can be merged, it will first need to pass continuous
+  integration tests and be reviewed. Sometimes the review process can be slow,
+  so please be patient.
```
Comment (Eric-Arellano) [inline qiskit_bot.yaml:8]: Abby M wanted an emoji at the end hah. Maybe this?

```suggestion
  so please be patient. Thanks! 🙌
```

**Review by Eric-Arellano — APPROVED** [review]
Nice!

**Review by arnaucasau — APPROVED** [review]
Awesome work! Thank you Frank!

File: qiskit_bot.yaml [hunk 2]
```diff
@@ -1,30 +1,240 @@
...
-# We use quotes around users who don`t want a GitHub notification,
-# but whom we still want their name in the Qiskit Bot message so
-# people know they are relevant.
+always_notify: true
+notification_prelude: |
+  Thanks for contributing to Qiskit documentation!
+
+  Before your PR can be merged, it will first need to pass continuous
+  integration tests and be reviewed. Sometimes the review process can be slow,
+  so please be patient. Thanks! 🙌
```
[the remark this answers is not in the document]
Comment (frankharkins) [reply qiskit_bot.yaml:8]: Good call! It's weird that github markdown ignores linebreaks in `.md` documents but not in issue comments 😕

**Pull request merged**